include/sdf/Element.hh

```diff
@@ -149,6 +149,12 @@ namespace sdf
     /// \param[in] _prefix String value to prefix to the output.
     public: void PrintValues(std::string _prefix) const;
 
+    /// \brief Output Element's values to stdout.
+    /// \param[in] _prefix String value to prefix to the output.
+    /// \param[in] _includeDefaults flag for default elements and attributes.
+    public: void PrintValues(std::string _prefix,
```

Comment:
We should pass in the `std::string` as a `const std::string &`

Reply:
addressed on https://github.com/osrf/sdformat/pull/575/commits/c9780654c9ba09a09d250b5776141a0dcd428b80